package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.3.3",
   "description": "A set of utilities for building Redux applications in Google Chrome Extensions.",
   "main": "lib/index.js",
+  "typings": "./index.d.ts",
```

Comment:
Can we place `index.d.ts` in the `lib` dir as well?

Reply:
Babel won’t transpile `.d.ts` file so it’s up to you if you want to add `cp` to the build script. Most other projects I see just put it under the root folder.

Reply:
If most put it under the root folder, let's keep it that way. As you mentioned, redux seems to do it that way, so I'm cool with it.